ga4gh/client.py

```diff
@@ -278,6 +296,35 @@ def searchVariantAnnotations(
             request, "variantannotations",
             protocol.SearchVariantAnnotationsResponse)
 
+    def searchFeatures(
+            self, featureSetId=None, parentId=None, referenceName=None,
+            start=0, end=None, featureTypes=[]
+                       ):
+        """
+        Returns the result of running a searchFeatures method
+        on a request with the passed-in parameters.
```

Comment:
I think you need to have a blank line between the summary and sphinx doc parameters. (Could be wrong though).